interface/forms/newpatient/common.php

```diff
@@ -76,444 +76,459 @@ function sensitivity_compare($a, $b)
+                                                            text(substr($irow['title'], 0, 40)) . "</option>\n";
+                                                        }
+                                                        ?>
+                                                    </select>
+                                                </div>
+                                                <div class="col-sm-12">
+                                                    <p><i><?php echo xlt('To link this encounter/consult to an existing issue, click the '
+                                                    . 'desired issue above to highlight it and then click [Save]. '
+                                                    . 'Hold down [Ctrl] button to select multiple issues.'); ?></i></p>
+                                                </div>
+                                            </div>
+                                        <?php
+                                        }
+                                        ?>
+                                    </div>
+                                </div>
+                            </div>
+                    </fieldset>
+                    <?php //can change position of buttons by creating a class 'position-override' and adding rule text-align:center or right as the case may be in individual stylesheets ?>
+                    <div class="form-group clearfix">
+                        <div class="col-sm-12 text-left position-override">
+                            <button type="button" class="btn btn-default btn-save" onclick="top.restoreSession(); saveClicked(undefined);"><?php echo xlt('Save');?></button>
+                            <?php if ($viewmode || !isset($_GET["autoloaded"]) || $_GET["autoloaded"] != "1") { ?>
+                                <button type="button" class="btn btn-link btn-cancel btn-separate-left" onclick="top.restoreSession(); location.href='<?php echo "$rootdir/patient_file/encounter/encounter_top.php";?>';"><?php echo xlt('Cancel');?></button>
+                            <?php } else { // not $viewmode ?>
+                            <button class="btn btn-link btn-cancel btn-separate-left link_submit" onClick="return cancelClicked()">
+                                    <?php echo xlt('Cancel'); ?></button>
+                            <?php } // end not $viewmode ?>
+                        </div>
+                    </div>
+                    <div class="clearfix"></div>
+                </form>
+            </div>
+        </div>
+    </div><!--end of container div-->
+     <br>
+    <?php
+    //home of the help modal ;)
+    //$GLOBALS['enable_help'] = 0; // Please comment out line if you want help modal to function on this page
+    if ($GLOBALS['enable_help'] == 1) {
+        echo "<script>var helpFile = 'common_help.php'</script>";
+        //help_modal.php lives in interface, set path accordingly
+        require_once "../../help_modal.php";
```

Comment:
In the odd case where you bring in help_modal twice, it should be able to do that, correct? If so, then would remove the once and make it just `require`

Reply:
globals.inc.php entry to allow for 3 states for the modal 

````
array(
               '0' => xl('Hide Help Modal'),
               '1' => xl('Show Help Modal'),
               '2' => xl('Disable Help Modal'),
           ),
````